### ISSUE TITLE
[FR] Update Firmware without exposing Shelly‘s to the internet

### ISSUE THREAD

**Issue by Cirrus94** [StyraHem/ShellyForHASS#461]
## Is your feature request related to a problem? Please describe.

I am trying to block as many smart home devices from connecting to the internet. Everytime when i wan‘t to update my Shelly‘s i have to unblock them. Even if I just want to know if there is a new version available i have to unblock at least one of each type. 
## Describe the solution you'd like

It would be great to download firmware updates over the Shelly component in Home Assistant without having to unblock each device from connecting to the internet.

**Comment by hakana** [comment]
You mean only expose HA to internet?

**Comment by Cirrus94** [comment]
> You mean only expose HA to internet?

Yes exactly. The only connection to the „outside“ should be my HA server. I think many other people aim for the same goal.
